types/types.go

```diff
@@ -671,6 +671,9 @@ type SystemContext struct {
 	CompressionFormat *compression.Algorithm
 	// CompressionLevel specifies what compression level is used
 	CompressionLevel *int
+	// Give priority to pulling zstd images if multiple images are present unless explicitly
+	// configured false.
+	GivePriorityToZstdImages OptionalBool
```

Comment:
Just to explicitly repeat this, this should be controlled in `copy.Options`, not in `types.SystemContext`.